influxql/iterator.go

```diff
@@ -882,6 +882,16 @@ func (opt IteratorOptions) ElapsedInterval() Interval {
 	return Interval{Duration: time.Nanosecond}
 }
 
+// IntegralInterval returns the time interval for the integral function.
+func (opt IteratorOptions) IntegralInterval() Interval {
```

Comment:
This looks slightly different from `DerivativeInterval()`. Is there a reason this is different?